### DISCUSSION TITLE
I need to end conversation when I push new chat button in chat completion

### DISCUSSION THREAD

**Discussion by zicas2000** [srcnalt/OpenAI-Unity#49] [Q&A] [answered]
Thanks alot for this amazing package, Is there any way to terminate the request and establish a new one with chat completion endpoint?

**Comment by srcnalt** [comment, marked as answer]
Hi @zicas2000, the chat messages are a List that is being sent to the ChatCompletion endpoint, you can simply clean the list when you end the conversation. And every new conversation inject your initial prompt as a first message before sending a new one if that is necessary.

**Reply by zicas2000** [reply]
Thanks so much. @srcnalt
